src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -74,6 +79,8 @@ public function ask(InputInterface $input, OutputInterface $output, Question $qu
      */
     public function setInputStream($stream)
     {
+        @trigger_error(sprintf('The setInputStream() method is deprecated since version 3.1 and will be removed in 4.0. Use %s:setStream() instead.', StreamableInputInterface::class), E_USER_DEPRECATED);
```

Comment:
You can also use the `__METHOD__` magic constant instead of `setInputStream` to be consistent with other deprecation triggers.